brain_games/cli.py

```diff
@@ -33,3 +36,12 @@ def loose(answer: str, correct_answer: str, user_name: str) -> None:
 
 def congratulate(user_name: str) -> None:
     print(f"Congratulations, {user_name}!")
+
+
+def num_random(bottom: int, ceiling: int) -> int:
+    return randint(bottom, ceiling)
+
+
+def get_data(d1: str, d2: str, d3: str) -> Dict[str, str]:
+    task_params = {"rules": d1, "question": d2, "correct_answer": d3}
+    return task_params
```

Comment:
these functions aren't for cli (as they're not working with cli), but rather utils for `games` package. move them into `games/utils.py` module